inc/RX.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * RX.h
+ *
+ *  Created on: Oct 6, 2018
+ *      Author: larsonma
+ */
+
+#ifndef RX_H_
```

Comment:
I know this would be a lot of backtracking, but could we call it something like DATA instead of RX now? I called it RX because I thought we would have a transmitter line and a separate receiver line. To get ahead, I would want to wait until we get ahead with the transmitter milestone before we change it if we decide to change it.